src/fgimic/vtkplot.f90

```diff
@@ -311,5 +311,83 @@ subroutine write_vtk_vector_unstructuredgrid(fname, grid, pdata, cells)
         call closefd(fd)
     end subroutine
 
+    ! fname: output filename
+    ! grid: grid point coordinates
+    ! pdata: vectors at grid points
+    ! cell definitions
+    subroutine write_vtk_imagedata_unstructuredgrid(fname, grid, pdata, cells)
```

Comment:
I think I'd call this 'write_vtk_scalar_unstructuredgrid'.

Reply:
hm, this is a matter of taste. For the structured grid we called it also "image data".

Reply:
to me, we are printing a scalar field (which also applies to the comment one up).  This is in analogy to the vector field that is printed in the other function that is called 'write_vtk_vector_unstructuredgrid'.

Reply:
I changed the name in my last commit.